src/utils.ts

```diff
@@ -9,3 +11,14 @@ export function debounce(fn, wait = 0) {
         }, wait);
     };
 }
+
+export const routerDebounce = 75;
```

Comment:
Why 75? Does it not work just with min debounce?

Reply:
yh it just seemed to be consistent on my machine